tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,10 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    
+    # more pythonic than assert len(items) == 0
+    assert not items
```

Comment:
While it's true that (as in your comment) this is a more pythonic check in general, here in this test, I would actually prefer the explicit `len()` check, or even comparing to a literal `[]`. The reason is that the function is supposed to return a list, and I like my tests to actually check that the thing being returned is actually what was expected, while here, this just tells us that something falsy was returned.

So in regular python code, go with the implicit checks, but in a test, I want to be sure of the actual thing being returned.